### ISSUE TITLE
composer update fails due to "Clear Twig caches on deploys" patch

### ISSUE THREAD

**Issue by danielnitschelc** [acquia/blt#1764]
The core patch "Clear Twig caches on deploys" in [composer.required.json](https://github.com/acquia/blt/blob/8.x/composer.required.json#L49) does not apply to Drupal 8.3.5, and so composer updates are broken:

```
Gathering patches for dependencies. This might take a minute.
  - Installing drupal/core (8.3.5): Loading from cache
  - Applying patches for drupal/core
    https://www.drupal.org/files/issues/no_reliable_method-2752961-29.patch (Clear Twig caches on deploys)
   Could not apply patch! Skipping. The error was: Cannot apply patch https://www.drupal.org/files/issues/no_reliable_method-2752961-29.patch

                                                                                                                              
  [Exception]                                                                                                                 
  Cannot apply patch Clear Twig caches on deploys (https://www.drupal.org/files/issues/no_reliable_method-2752961-29.patch)!  
```

**Comment by danielnitschelc** [comment]
To temporarily fix, add this to your local `composer.json` (just under `require-dev`):
```
    "conflict": {
        "drupal/core": "8.3.5"
    },
```

**Comment by grasmash** [comment]
Resolved by #1761
